centos_ci.sh

```diff
@@ -255,10 +261,39 @@ function setup_build_environment() {
   runuser -l minishift_ci -c "/bin/bash centos_ci.sh"
 }
 
-function perform_release() {
-  setup_kvm_docker_machine_driver; # Required for integeration tests
-  cd gopath/src/github.com/minishift/minishift
+#############################################################################
+# Perform the publishing of Minishift documentation after integrating it with
```

Comment:
@minishift/minishift-dev Does this comment make sense? 

Check the argument section. Is it conveying that 1 argument is Repo owner, 2nd is Branch name etc

Reply:
@budhrg  How about *Publish Minishift documentation after integrating it with Openshift documentation*?

Reply:
Not sure what you mean... but I miss checking for empty values. If any of the arguments is missing, it should fail